e2e/test_events.py

```diff
@@ -138,6 +143,55 @@ def clean_up():
     clean_up()
 
 
+@prefix('test_events')
+def test_invalid_event_swap(team, user, role):
+    team_name = team.create()
+    team_name_2 = team.create()
+    user_name = user.create()
+    user_name_2 = user.create()
+    role_name = role.create()
+    user.add_to_team(user_name, team_name)
+    user.add_to_team(user_name_2, team_name)
+    user.add_to_team(user_name_2, team_name_2)
+
+    start, end = int(time.time()) + 100, int(time.time() + 36000)
+
+    def clean_up():
```

Comment:
Use the event fixture to get around the clean_up() calls. Only reason I didn't use it in test_events() is to make sure testing events POST isn't implicitly tied to the implementation of the fixture.